[REF] build_image.sh: change pg log_directory

Angelfentanez opened the pull request:
This PR is required because now (in some cases) we will allow the normal users to access the postgres log, so we need to move it to a path where we can manage the permissions without problems

Angelfentanez commented:
@ruiztulio PR ready to be reviewed

ruiztulio commented:
@Angelfentanez add the env var ```PG_LOG_PATH``` in the global bash configuration so it is available always and do the same changes for [16.04](https://github.com/Vauxoo/docker-ubuntu-base/tree/16.04) please

Angelfentanez commented:
@ruiztulio Done, I added the `PG_LOG_PATH` env var in the global bash and changed the permissions of the pg_log directory so postgres can create the logs there, [this](https://github.com/Vauxoo/docker-ubuntu-base/pull/35) is the PR for the branch 16.04

ruiztulio commented:
@moylop260 do you think we can have issues with the big image because of this change? I already merged the PR to the 16.04 because it is not used as base for the odoo_sippable one.
